src/App.tsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import {Header} from './components/Header/Header';
+import {MovieContainer} from './components/MovieContainer/MovieContainer';
+import {Footer} from './components/Footer/Footer';
+import {Logo} from './components/Logo/Logo';
+
+export const App = () => (
+    <>
+        <React.StrictMode>
+            <Header />
+            <MovieContainer />
+            <Footer children={<Logo />} />
```

Comment:
It's better to use this form:
```js
<Footer>
  <Logo />
</Footer>
```

But personally for this case i don't see much sense to use `children`, you can just import `Logo` in `Footer` component